src/Microsoft.ML.LightGBM/LightGbmCatalog.cs

```diff
@@ -46,14 +46,14 @@ public static class LightGbmExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/>.
+        /// Predict a target using a boosted decision tree regression model trained with the <see cref="LightGbmRegressorTrainer"/> and advanced options.
         /// </summary>
         /// <param name="catalog">The <see cref="RegressionCatalog"/>.</param>
-        /// <param name="options">Advanced options to the algorithm.</param>
+        /// <param name="options">Trainer options.</param>
```

Comment:
>Trainer [](start = 34, length = 7)

either estimator or algorithm

Reply:
I've been changing them to "Trainer Options" based on Rogan's comment on AveragedPercentron. I don't want to change back everything. Trainer sounds good enough,  unless there's a strong objection. 

---
In reply to: [263689283](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263689283) [](ancestors = 263689283)

Reply:
I think this is preferred to "advanced".

---
In reply to: [263905459](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263905459) [](ancestors = 263905459,263689283)